Sources/SATSCore/Components/WheelPickerView/WheelPickerView-Config.swift

```diff
@@ -0,0 +1,73 @@
+import CoreGraphics
+
+public extension WheelPickerView {
+    struct WheelConfig {
+            /// The minimum value the picker will allow
+        let minValue: CGFloat
```

Comment:
there is something off with the indentation here

Reply:
Yeah, I just saw! Autoformatted the file and didn't see that. 👍